### ISSUE TITLE
Returned API Object no longer serializable

### ISSUE THREAD

**Issue by McIntozh** [heidelpay/heidelpayJava#75]
Starting with version 1.8.0.0 it is no longer possible to serialize the objects retured from the API into JSON using GSON for example.
This is because the transient modifier in the class AbstractPayment for the field Payment (and perhaps the other fields) has been removed, thus leading to a StackOverflowError during serialization.
Is this intended behaviour?

Example code working with 1.7.0.0 but not with 1.8.0.0:
```
  public static void main(String[] args) throws HttpCommunicationException {
    Heidelpay heidelpay = new Heidelpay("p-priv-1234567");
    Charge charge = heidelpay.fetchCharge("p-pay-123", "p-chg-1");
    System.out.println(new Gson().toJson(charge));
  }
```

**Comment by sadrian** [comment]
We just released a new version 1.9.1.0, where the problem should be fixed - pls try out.

**Comment by DerAusHeidelberg** [comment]
Hi @McIntozh,

we have now released a Fix for this Issue in Version 1.11.2.2.

Please see https://search.maven.org/artifact/com.heidelpay.payment/heidelpayJava/1.11.2.2/jar

Thank you for helping us to improve.
I will now close this Issue.
